File: src/web-ui/programmer-webui/src/app/views/treatment/models/TreatmentModel.ts
import { TreatmentStatus } from "./TreatmentStatus";

export class TreatmentModel {
    values: TreatmentValuesModel;
    id: string;
    status: TreatmentStatus;

    constructor() {
        this.values = null;
    }
}

export class TreatmentValuesModel {
    vtbi: number;
    rate: number;
    dose: number;
}

File: src/server/Application/Programmer.Common/Domain/Treatment/TreatmentModel.cs
namespace Programmer.Common.Domain.Treatment
{
    public class TreatmentModel:DomainModelBase<long>
    {
        public string SessionId;
        public decimal Vtbi { get; set; }
        public decimal Rate { get; set; }
        public decimal Dose { get; set; }
    }
}